python/cog/director/director.py

```diff
@@ -170,6 +174,9 @@ def _handle_message(self, message_id, message_json) -> None:
         )
         should_cancel = self.redis_consumer.checker(message.get("cancel_key"))
         prediction_id = message["id"]
+        span.set_attribute("prediction.id", prediction_id)
+        span.set_attribute("prediction.created_at", message.get("created_at", None))
+        span.set_attribute("prediction.started_at", message.get("started_at", None))
```

Comment:
`started_at` won't be set in the message, you'll need to get it from the tracker. For ease and consistency, it might be better to add all these attributes after the tracker is created, by just pulling them from the tracker.